react/CozyTheme/Readme.md

```diff
@@ -0,0 +1,45 @@
+An area of a page can have a different CozyTheme and inside components
+will be automatically styled.
+
+- At the moment only some Buttons support this.
+
+
+```
+import CozyTheme from 'cozy-ui/transpiled/react/CozyTheme';
+import { Title, SubTitle } from 'cozy-ui/transpiled/react/Text';
+import Button from 'cozy-ui/transpiled/react/Button';
+import BarButton from 'cozy-ui/transpiled/react/BarButton';
+
+const props = [{}, { disabled: true}, { busy: true }];
+
+const themesSupportingContext = [
+  'regular',
+  'secondary'
+];
+
+<CozyTheme variant='inverted' className='u-p-1'>
+  <Title className='u-white'>Inverted theme</Title>
+  <SubTitle className='u-white'>Buttons</SubTitle>
+  {themesSupportingContext.map(theme =>
+    <p key={theme}>{
+      props.map(
+        props => <Button key={theme + JSON.stringify(props)} label={theme} theme={theme} {...props} />
```

Comment:
i'm not sure this is a problem, but I was initially confused about Cozy theme vs Button theme by reading this example. Seeing it live makes it clearer though, but I wonder if we should make that distinction in the code?

Reply:
Do you have a suggestion here ?

Reply:
I do not plan to make any change here as I want to merge this PR ASAP.

Reply:
No change needed, it's more of a nit. I was mostly interested to know if you or someone else thinks the 2 theme concepts here is confusing or not. We can improve this as the example grows.

Reply:
Yes I think this is a bit confusing.